src/BitStringAddresses/occupationnumberfs.jl

```diff
@@ -66,7 +66,14 @@ end
 Base.reverse(ofs::OccupationNumberFS) = OccupationNumberFS(reverse(ofs.onr))
 
 onr(ofs::OccupationNumberFS) = ofs.onr
-Base.isless(a::OccupationNumberFS, b::OccupationNumberFS) = isless(b.onr, a.onr)
+function Base.isless(a::OccupationNumberFS{M}, b::OccupationNumberFS{M}) where {M}
+    # equivalent to `isless(reverse(a.onr), reverse(b.onr))`
```

Comment:
why is it defined "backwards"?

Reply:
If you compare two `BoseFS`, the comparison is done on the bitstrings directly and we store the addresses in reverse order. This makes the ordering on `OccupationNumberFS` the same as that of `BoseFS`.

It might make more sense to sort all addresses lexicographically by ONR instead, but that would be a bigger change.